src/client/components/Swipe/Swipe.jsx

```diff
@@ -70,7 +99,13 @@ class Swipe extends Component {
         if (response.data.length == 0) {
           instance.setState({ hasNext: false });
         }
-        instance.setState({ places: response.data });
+        console.log(response.data)
```

Comment:
remove this

Reply:
Have not actually reviewed the code, but pulled and tested the branch. Seems to work alright except for the bug you mentioned,

Reply:
Bug fixed!